tests/test_generated_examples.py

```diff
@@ -770,7 +770,7 @@ def test_applicationfee_retrieve(self, request_mock):
             "/v1/application_fees/fee_xxxxxxxxxxxxx",
         )
 
-    def test_applicationfee_feerefund_list(self, request_mock):
+    def test_applicationfee_applicationfeerefund_list(self, request_mock):
```

Comment:
Is this change expected too?

Reply:
Yes, see the codegen PR for an explanation.